yb-voyager/cmd/importData.go

```diff
@@ -933,12 +933,19 @@ func executeSqlFile(file string) {
 
 	var errOccured = 0
 	sqlStrArray := createSqlStrArray(file, "")
-	for _, sqlStr := range sqlStrArray {
+	retryCount := INDEX_RETRY_COUNT
+	for i := 0; i < len(sqlStrArray); {
+		sqlStr := sqlStrArray[i]
 		log.Infof("Run query %q on target %q", sqlStr[1], target.Host)
 		_, err := conn.Exec(context.Background(), sqlStr[0])
```

Comment:
Let's define a separate function `execSQLStatementWithRetries(conn, sql)`. Move the call to the `conn.Exec()` in the new function. The function should take care of retries internally. That should leave the `for _, sqlStr := range sqlStrArray` loop intact.

Reply:
done